src/coreml.cs

```diff
@@ -381,5 +425,131 @@ interface MLCustomLayer {
 		[Export ("encodeToCommandBuffer:inputs:outputs:error:")]
 		bool Encode (IMTLCommandBuffer commandBuffer, IMTLTexture[] inputs, IMTLTexture[] outputs, [NullAllowed] out NSError error);
 	}
+
+	[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+	[BaseType (typeof(NSObject))]
+	interface MLArrayBatchProvider : MLBatchProvider {
+
+		[Export ("array")]
+		IMLFeatureProvider[] Array { get; }
+
+		[Export ("initWithFeatureProviderArray:")]
+		IntPtr Constructor (IMLFeatureProvider[] array);
+
+		[Export ("initWithDictionary:error:")]
+		IntPtr Constructor (NSDictionary<NSString, NSArray> dictionary, [NullAllowed] out NSError error);
```

Comment:
since 
* it's not mutable - no setter on `Array`; and
* none of the `.ctor` accept a `null` input 

then it seems unlikely that `init` is allowed - so the type should be decorated with `[DisableDefaultCtor]`